drivers/ipm/Kconfig

```diff
@@ -82,3 +82,19 @@ config IPM_MHU
 	depends on IPM
 	help
 	  Driver for SSE 200 MHU (Message Handling Unit)
+
+config IPM_STM32_IPCC
+	bool "STM32 IPCC controller"
+	default n
```

Comment:
Redundant `default n`. See [this page](https://docs.zephyrproject.org/latest/guides/kconfig/index.html#redundant-defaults).